README.md

```diff
@@ -14,14 +14,10 @@ A **development container** is a running [Docker](https://www.docker.com) contai
 This repository contains a set of **dev container templates** which are source files packaged together that encode configuration for a complete development environment. A template can be used in a new or existing project, and a [supporting tool](https://containers.dev/supporting) will use the configuration from the template to build a development container.
 
 ⚠️ Development container 'templates' are a
```

Comment:
At this point, should we just remove this note?  We wouldn't make breaking changes at this point I think.

Reply:
Makes sense!